articles/azure/azs-how-create-scale-set.md

```diff
@@ -0,0 +1,126 @@
+
+    - **Username** – Admin username for the virtual machines.
+
+    - **Password** – Admin password for the virtual machines.
+
+    - **Confirm password** – Confirm the password.
+
+    - **Instance count** – The number of virtual machines in the scale set (0 - 100).
+
+    - **Instance size** – The size of each virtual machine in the scale set. Some sizes may only be available using templates, PowerShell, or Azure CLI due to the recommended portal defaults.
+
+    - **Use managed disks** – Managed disks is a feature that simplifies disk management for Azure laaS virtual machines by handling storage account management for you. For more information, see the [Managed Disks overview](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview)
+
+    - **Public IP address name** – The name of the public IP address for the load balancer in front of the scale sets.
+
+    - **Public IP allocation method** – Select between Dynamic or Static IP addresses.
+
+    - **Domain name label** – Domain name label for the load balancer in front of the scale set.
+
+    - **Virtual network** – This is the virtual network you would like the scale set to be connected to.
+
+    - **Public IP address per instance** – Add a public IP address to each instance in the scale set.
+
+    ![BASICS page](images/azs-browser-create-scale-set.png)
+
+6. You can monitor the progress of your scale set's deployment by clicking the **Notifications** icon.
+
+    ![Deployment in progress notification](images/azsp_createvm_progress.png)
+
+7. When the deployment is finished, the notification will change to *Deployment succeeded*.
+
+    ![Deployment complete](images/azsp_createvm_deployment_success.png)
+
+8. After the scale set has deployed, you can view it by clicking **All Services** in the favourites panel, then select **Virtual machine scale sets** from the *compute* section.
+
+    ![All resources menu](images/azs-browser-select-all-services.png)
+
+9. Select your scale set from the list.
+
+    ![All resources list](images/azs-browser-select-scale-set.png)
+
+10. In the blade for your scale set, you can view and monitor the scale set, change its settings and perform diagnostics and troubleshooting.
+
+    ![scale set overview](images/azs-browser-Scale-Set-overview.png)
+
+## Modifying the instance count of your scale set
+
+> [!Warning]
+> Currently auto-scaling is not supported on Azure Stack.
+
+1. In the blade for the scale set, select **Scaling** under *settings*.
+
+    ![scale set scaling button](images/azs-browser-button-scaling.png)
+
+2. Manually increase or decrease the number of instances within the scaling settings using the slide bar.
+
+    ![Scaling slide bar](images/azs-browser-instance-count-slidebar.png)
+
+3. Once you have selected the desired number of instances select **Save** to commit the change.
+
+    ![Save number of instances for the scale set](images/azs-browser-button-save-instance-count.png)
```

Comment:
The edges of the oval around the Save button are cropped. Could you edit to include all of the oval?